src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo___link" href="#home">
+        <img src="images/logo.png" alt="MOYO">
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#Apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Laptops & Computers"
+              data-qa="hover"
+            >
+                Laptops & Computers
```

Comment:
```suggestion
            <a
              class="nav__link"
              href="#Laptops & Computers"
              data-qa="hover"
            >
              Laptops & Computers
```
there should be 2 indents